cpu/stm32f4/periph/cpuid.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2014 FU Berlin
```

Comment:
Best practice is to use one commit per module i.e. CPU in this case and add commits with review changes prefixed by `SQUASHME` or `FIX`. These have be squashed when ACKed before merging.